packages/core/bootstrap/src/lib/ws/epics.ts

```diff
@@ -33,6 +38,8 @@ import {
   ws_subscription_total,
 } from './metrics'
 
+const debug = (message: any, withParams = false) => tap((value: any) => withParams ? console.log(message, value) : console.log(message))
```

Comment:
Should we be using the app provided logger rather than console log?

Reply:
This was mostly for this PR, I don't think it will stay after is Ready to merge